ods_ci/tasks/Resources/RHODS_OLM/uninstall/codeflare_uninstall.resource

```diff
@@ -24,6 +24,13 @@ Uninstall CodeFlare
     ...    oc delete mcads.codeflare.codeflare.dev mcad -n redhat-ods-applications --ignore-not-found
     Should Be Equal As Integers	${return_code}	 0   msg=Error deleting CodeFlare MCAD CR
     Uninstall ISV Operator From OperatorHub Via CLI     rhods-codeflare-operator    openshift-operators
+    Uninstall CodeFlare CatalogSource
+
+Uninstall CodeFlare CatalogSource
+    [Documentation]    Removes the CodeFlare cluster-wide catalog source created for managed env, if it exists
+    ${return_code}    ${output}    Run And Return Rc And Output
+    ...    oc delete catalogsource rhods-codeflare-catalog -n openshift-marketplace --ignore-not-found
```

Comment:
I would suggest you to delete the subscription of codeflare before deleting the catalog source

Reply:
Sure, good point!

Reply:
@tarukumar Not sure if I miss something, but it seems to me that the catalogsource is already being deleted after the subscription is removed? Please see the lines https://github.com/red-hat-data-services/ods-ci/pull/951/files#diff-f757092a94298927c14b51bf57cfd6bdc2bc6779d5690ef73482aa2a5d35f29aR26-R27

Reply:
correct, i overlooked